packages/server/modules/core/repositories/users.ts

```diff
@@ -51,16 +51,30 @@ export async function getUsers(
   userIds = isArray(userIds) ? userIds : [userIds]
 
   const q = Users.knex<UserWithOptionalRole[]>().whereIn(Users.col.id, userIds)
+  q.leftJoin(
+    USER_EMAILS_TABLE_NAME,
+    `${USER_EMAILS_TABLE_NAME}.userId`,
+    Users.col.id
+  ).where({ primary: true })
```

Comment:
If you use the DB table helper, all column names are always prefixed with an alias/table name to avoid ambiguity about which table does the column belong on. There may only be 1 `primary` col now, but that's some nice future proofing you get out of the box.